sys/include/net/gnrc/netif/internal.h

```diff
@@ -333,7 +333,7 @@ static inline bool gnrc_netif_is_rtr_adv(const gnrc_netif_t *netif)
  * @brief   Checks if the interface uses a protocol that requires 6Lo to run
  *
  * @attention   Requires prior locking
- * @note        Assumed to be true, when @ref GNRC_NETIF_NUMOF == 1 and
+ * @note        Assumed to be true, when @ref gnrc_netif_single return true and
```

Comment:
Independent of the name

```suggestion
 * @note        Assumed to be true, when @ref gnrc_netif_single() return true and
```